meganav.yaml

```diff
@@ -832,7 +838,7 @@ get_ubuntu:
               secondary_cta_title: Get Ubuntu for Raspberry Pi
               secondary_cta_url: /download/raspberry-pi
             - title: Ubuntu Core for Raspberry Pi
-              description: Embedded OS
+              description: Embedded Linux
```

Comment:
```suggestion
              description: Embedded OS
```
This still reads as such on the doc 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/c8010eeb-e646-4be4-8544-734f761ed5fa)

Reply:
Copy doc was wrong in this case, thank you for flagging it up.